src/java.desktop/share/classes/sun/awt/FontConfiguration.java

```diff
@@ -852,11 +852,6 @@ private static Vector<String> splitSequence(String sequence) {
         return parts;
     }
 
-    protected String[] split(String sequence) {
-        Vector<String> v = splitSequence(sequence);
```

Comment:
Can `splitSequence` be replaced with `sequence.split(",")`. The created vector is used only to convert it to array. Using `split` returns the array.

https://github.com/openjdk/jdk/blob/7170385dd8028ad8c2c84b619bee0ade3af3ff09/src/java.desktop/share/classes/sun/awt/FontConfiguration.java#L2118

Reply:
Initially I was intended to do so (#11380), but I found that there is slight difference between `splitSequence` and `Strings.split` implementations. They handle empty _elements_ differently. So I think it's worth to create a separate issue to investigate if it's worth to change.

Reply:
Reasonable. Thanks.

As another suggestion, since `Vector` isn't returned, `splitSequence` could use `ArrayList` internally instead.